lms/djangoapps/courseware/module_render.py

```diff
@@ -642,6 +643,8 @@ def rebind_noauth_module_to_user(module, real_user):
 
     system.set(u'user_is_staff', user_is_staff)
     system.set(u'user_is_admin', has_access(user, u'staff', 'global'))
+    system.set(u'user_is_beta', CourseBetaTesterRole(course_id).has_user(user))
```

Comment:
The variable name should be `user_is_beta_tester`.

Can we follow the existing pattern of doing the actual check in `has_access` and just calling `has_access(user, 'beta_tester', course_descriptor, course_key)` from here?